### DISCUSSION TITLE
Welcome 🤩 - are you new here? 👉🏽 please drop a comment 🥳

### DISCUSSION THREAD

**Discussion by tippexs** [nginx/unit#986] [General]
Great to see all the people joining the Nginx Unit GitHub repo! I have already dropped some messages on Twitter.
We would love to hear your story and why you are here. Where did you hear about the project? Looking forward to all the interesting comments!
Timo and the Unit Team

**Comment by Pavlusha311245** [comment]
I learned about Nginx Unit from an article on Habr. https://habr.com/ru/articles/588785/
I am a Laravel developer and I wanted to optimize my work. This was in 2020 or 2021. At that time I was in the process of containerizing an application in Docker.   One of the first queries led to the article nginx + php-fpm. I did not like this bundle as it was heavy and used nginx as a proxy. The article was a comparison of different web servers and there were two winners. Nginx Unit and Roadrunner. I was interested in the new web server and when I found out that it uses JSON for configuration I wanted to try it. I was also impressed by the availability of modules and in particular PHP module to run the application I needed.
That's the story :)

**Comment by jameshulse** [comment]
Hi. I also happen to be a Laravel developer and came across Nginx Unit when researching simpler alternatives to Nginx. I really enjoyed setting up Unit but unfortunately went back to full Nginx for the gzip support. I will be back though once there is a story to get that working directly.

**Comment by MarkusEicher** [comment]
Hello Nginx Unit team and hi community. My name is Markus and it's nice to meet you all. 👋 

I learned about Nginx Unit about a few months ago, while doing evaluation of webserver / proxy / application server options. I finally found my favorite with Unit. That's why I'm here. Eager to learn and interact with you and I hope that I can contribute at least a bit to the success of this project. 

You folks from the Nginx Unit team do a great job. I like the product so far after a first quick test run. What I appreciate a lot, is your community oriented opensource approach. Currently started building a completely new development environment and CI-CD workflow for Machine Learning and AI workloads and solutions based on WASM and WASI-NN (WASI Preview 2 for components as soon as its available). 

This almost cries for using Nginx as web and app server. Need to get my hands dirty and build it. Sure I will love to interact on GitHub and with the community. I wish you much success with the further development of Nginx Unit. Let's do this. 🚀
